src/llama_stack_client/lib/cli/models/get.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+import argparse
+import json
+
+from llama_stack_client import LlamaStackClient
+from llama_stack_client.lib.cli.configure import get_config
+from llama_stack_client.lib.cli.subcommand import Subcommand
+
+from tabulate import tabulate
+
+
+class ModelsGet(Subcommand):
+    def __init__(self, subparsers: argparse._SubParsersAction):
+        super().__init__()
+        self.parser = subparsers.add_parser(
+            "get",
+            prog="llama-stack-client models get",
+            description="Show available llama models at distribution endpoint",
+            formatter_class=argparse.RawTextHelpFormatter,
+        )
+        self._add_arguments()
+        self.parser.set_defaults(func=self._run_models_list_cmd)
+
+    def _add_arguments(self):
+        self.parser.add_argument(
+            "model_id",
+            type=str,
+            help="Model ID to query information about",
+        )
+
+        self.endpoint = get_config().get("endpoint")
+        self.parser.add_argument(
+            "--endpoint",
```

Comment:
So I think @raghotham wanted things to work like the `aws` CLI. i.e., there's a notion of a "profile"... its default value is "default". And you can update the default. And then each command gets an optional "profile" name which it uses to look up the corresponding YAML and gets all its values from there. 

Specifically, we should never have an endpoint here (and in all commands) but only a "name" which represents a link to the YAML.

Reply:
Hmm, my understanding is that for now we only have endpoint to store in "profile", but I could remove this `--endpoint` s.t. we read from "profile" config only.